autopush/websocket.py

```diff
@@ -1090,7 +1094,8 @@ def ver_filter(update):
             size = len(msg.data) if msg.data else 0
             log.msg("Ack", router_key="webpush", channelID=chid,
                     message_id=version, message_type="direct",
-                    message_size=size, uaid_hash=self.ps.uaid_hash)
+                    message_size=size, uaid_hash=self.ps.uaid_hash,
+                    **{'user-agent': self.ps.user_agent})
```

Comment:
Oh, my bad, ugh, we use underscore and dash randomly. How annoying. Let's file a separate bug to have a code-wide consistency to all logging keys cause this is getting out of hand already.